components/LinkButton/LinkButton.jsx

```diff
@@ -1,31 +1,24 @@
 import PropTypes from 'prop-types';
 import Router from 'next/router';
 
-const LinkButton = ({ label, route }) => (
-  <div>
+const LinkButton = ({ label, route }) => {
+  const handleLink = (url) => window.open(url, '_blank');
+  const isGoogleForm = route && route.includes('google');
+  return (
     <button
       href="#"
       role="button"
       draggable="false"
-      className="govuk-button govuk-button--start govuk-!-margin-top-3"
+      className={`govuk-button govuk-link-button govuk-!-margin-bottom-3`}
       data-module="govuk-button"
-      onClick={() => Router.push(`${route}`)}
+      onClick={() =>
+        isGoogleForm ? handleLink(route) : Router.push(`${route}`)
```

Comment:
We are mixing business logic with presentational logic.
Can we have a prop like `isExternal` driving it we want to open the link in an external page?